settings/installer/ospd/ospd.spec

```diff
@@ -45,6 +45,9 @@ subparsers:
                     type: Value
                     help: Specifies whether ths SSL should be used for overcloud
                     default: 'no'
+                overcloud-hostname:
```

Comment:
I think that "features" might need to move to a different structure.. I mean, more and more features are going to be added so I think we should have a distinction between a "deployment" need (such as building a LB for example) and a feature of OSP-d, which might "pollute" our .spec file.
I think we need to think about this further, maybe moving it to a configuration file? (thinking out loud)